### ISSUE TITLE
Test and Manual Synthesis do not match

### ISSUE THREAD

**Issue by JackTemaki** [carpedm20/multi-speaker-tacotron-tensorflow#59]
Hello,

i just trained a model on the LJSpeech_1_1 dataset, and when i manually synthesize a sentence that was also part of the test set during training, i get slightly different results:

The original test alignment:
![test-step-000031000-align000](https://user-images.githubusercontent.com/10742135/50913804-79c0a000-1435-11e9-9644-d19ebda1f5e3.png)

The alignment when using synthesize.py:
![2019-01-09_17-15-55 manual](https://user-images.githubusercontent.com/10742135/50913856-99f05f00-1435-11e9-8b78-c58496fb65ed.png)

The later sound example also sounds worse in my opinion.

What could be the mistake i made?

best regards,
Nick

**Comment by JackTemaki** [comment]
Also, here are my commands and the hparams file:

`python3 train.py --data_path=datasets/LJSpeech_1_1 --load_path=logs/LJSpeech_1_1_2018-12-17_15-04-28`

`python3 synthesizer.py --load_path=logs/LJSpeech_1_1_2018-12-17_15-04-28 --text "noyes was seized without difficulty, as it was a part of the scheme that he should act as the dupe, and remain on the spot in london till all the money was obtained."  --is_korean=False`


[hparams.py.txt](https://github.com/carpedm20/multi-speaker-tacotron-tensorflow/files/2741782/hparams.py.txt)
